src/main/resources/io/jenkins/plugins/aws/global_configuration/AwsManagementLink/index.groovy

```diff
@@ -15,7 +15,6 @@ def st=namespace("jelly:stapler")
 l.layout(norefresh:true, permission:app.ADMINISTER, title:my.displayName, cssclass:request2.getParameter('decorate')) {
     l.main_panel {
         h1 {
-            l.icon(src: "${resURL}/plugin/aws-global-configuration/images/aws.png", class: 'icon-xlg')
```

Comment:
Removing the icon completely for consistency with other sub-pages. Replacing it with the symbol looked horrible anyway due to the letters "AWS" being part of the SVG symbol (which was redundant with the text of the `h1` tag).